sop-website/docs/Data-Requests/Data-Requests.mdx

```diff
@@ -3,3 +3,75 @@ title: Data Requests
 id: Data Requests
 description:  An SOP for elements related to data requests
 ---
+
+# Awaiting Approval
+
+# Data Requests
+
+# Purpose
+The purpose of this correspondence is to specify the content, format, and timeline of the data CHoRUS sites should be contributing to this first upload to the CHoRUS data repository.
+
+# Procedures
+**Number of patients**: The count is based on hospital admissions, not on patients. A given patient may therefore contribute to several encounters.  We are requesting the preparation and upload of data from 25% of your total expected contribution to the CHoRUS database.
+
+**Encounter selection**: For this first upload, we are requesting that encounters be sampled randomly from all ICU admissions since January 2019.
+
+**Data to be uploaded**: The data to be uploaded will include structured and unstructured (notes and reports) EHR elements, images and waveforms for the entire duration of an inpatient encounter (hospitalization). The hospitalization, inpatient encounter, or visit occurrence must include one or more admissions to an intensive care unit. OHDSI recommends that the admission correspond to the initial contact with a healthcare system, even if this contact was initiated in a different hospital.
+
+The data elements to be included will be the Tier 1 data elements identified by CHoRUS investigators as of primary significance across all four domains of data [Broad DA SOP](https://chorus-ai.github.io/data_acq_SOP/docs/Data-Uploading/Broad%20SOP/).
+
+1. Structured EHR Data
+    >Structured EHR data should populate all OMOP Clinical Data Tables, except the NOTE, NOTE_NLP, and FACT_RELATIONSHIP  tables. All uploads are expected to be in text format, preferably .csv. Since some text 
+    fields contain commas, we suggest enclosing all text fields within quotes “”, in accordance with .csv standards. The use of a different delimiter, such as bar “|”, is also acceptable. We require sites to run 
```

Comment:
When listing inline technical elements (e.g. coding characters, variables, etc) I think it is good to wrap them in backticks (so instead of “|”, perhaps "`|`"). This helps to emphasize that they are technical terms and sometimes avoides formatting issues).